app.js

```diff
@@ -65,12 +61,12 @@ app.get('/', function (req, res) {
 app.get(/^\/(\w+)\/hello/, function (req, res) {
   var lang = req.params[0];
   // Just get this string
-  mybundle.getEntryInfo({ resourceKey: 'hello', languageId: lang}, function (err, data) {
+  mybundle.entry({ resourceKey: 'hello', languageId: lang})
+   .getInfo(function (err, entry) {
```

Comment:
Or put another way:
> var anEntry = mybundle.entry({ resourceKey: 'hello', languageId: lang});

`anEntry` is now a `ResourceEntry` _object_  It does not have any translation data in it. But you can call `.getInfo`…

> anEntry.getInfo(function (err, entryWithInfo) { … } );

… Now, `entryWithInfo` has the full information for the entry - the `value`, the `updatedAt` (mod time), etc.  Also, `entryWithInfo` is itself a `ResourceEntry` object, so  you could make further calls.